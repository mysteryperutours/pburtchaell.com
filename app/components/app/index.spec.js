import React from 'react';
import App from './index';

describe(App.name, () => {
  it('should render component', () => {
    expect(shallow(
      <App />
    )).toMatchSnapshot();
  });
});
