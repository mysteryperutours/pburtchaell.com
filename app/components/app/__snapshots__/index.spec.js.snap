// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRoot should render component 1`] = `
<div
  className="app-root"
/>
`;
